deploy-techdocs/action.yaml

```diff
@@ -24,9 +27,9 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    - uses: actions/setup-python@v3
```

Comment:
Resolved these issues:

<img width="760" alt="image" src="https://user-images.githubusercontent.com/423234/196923150-e2ab6b62-c410-4825-a518-5daa671c0dbd.png">

See also:
- https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/
- https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/